public/components/common/helpers/ppl_docs/overview.tsx

```diff
@@ -18,7 +18,7 @@ PPL is the primary language used with observability in OpenSearch, and it suppor
 - Group similar data points.
 - Caluculate statistics on your data.
 
-PPL is availble in OpenSearch Dashboards and in a standalone command-line tool. Within OpenSearch Dashboards, you can use <a href="{{https://opensearch.org/docs/latest/dashboards/query-workbench/}}">Query Workbench</a> to run on-demand PPL commands and view and save the results as text and JSON. The <a href="{{https://opensearch.org/docs/latest/search-plugins/sql/cli/}}">PPL command line interface (CLI)</a> is a standalone Python application that you can launch with the \`opensearchsql\` command. The PPL CLI is useful for running on-demand PPL commands and viewing and saving the results as text and JSON.
+PPL is availble in OpenSearch Dashboards and in a standalone command-line tool. Within OpenSearch Dashboards, you can use [Query Workbench](https://opensearch.org/docs/latest/dashboards/query-workbench/) to run on-demand PPL commands and view and save the results as text and JSON. The [PPL command line interface (CLI)] (https://opensearch.org/docs/latest/search-plugins/sql/cli/) is a standalone Python application that you can launch with the \`opensearchsql\` command. The PPL CLI is useful for running on-demand PPL commands and viewing and saving the results as text and JSON.
```

Comment:
Extra space in link, "[PPL command line interface (CLI)] (https://opensearch.org/docs/latest/search-plugins/sql/cli/)" vs "[PPL command line interface (CLI)](https://opensearch.org/docs/latest/search-plugins/sql/cli/)"

Reply:
Good catch. Thanks!